charm/charmcraft.yaml

```diff
@@ -60,7 +62,12 @@ requires:
     tracing:
         interface: tracing
         limit: 1
-
+    service-mesh:
+        limit: 1
+        interface: service_mesh
+    require-cmr-mesh:
```

Comment:
- Can we find a better name for the relation?
- Please add a `description:` field